docs/DarkSectorBattle.js.html

```diff
@@ -115,19 +115,19 @@ <h1 class="page-title">Source: DarkSectorBattle.js</h1>
      * The winner of the battle
      * @type {string}
      */
-    this.winner = data.DefId.$id === data.WinId.$id ? this.defender : this.attacker;
+    this.winner = data.DefId.$oid === data.WinId.$oid ? this.defender : this.attacker;
```

Comment:
to make it easier, we might make some instances for these variables we need to flex on for the time being.